SEDaily-IOS/HeaderView.swift

```diff
@@ -45,6 +47,22 @@ class HeaderView: UIView {
         super.init(frame: frame)
 
         self.performLayout()
+        
+        if UIDevice().userInterfaceIdiom == .phone {
+            switch UIScreen.main.nativeBounds.height {
```

Comment:
Probably not ideal to check by phone type, Can simplify to only check for iphone x. Eventually might want to look at a different way of doing our layout since overall it doesn't seem to  look as good on iphone x